src/mmw/js/src/analyze/views.js

```diff
@@ -8,20 +8,35 @@ var $ = require('jquery'),
     coreViews = require('../core/views'),
     chart = require('../core/chart'),
     windowTmpl = require('./templates/window.html'),
+    loadingTmpl = require('./templates/loading.html'),
+    errorTmpl = require('./templates/error.html'),
+    headerTmpl = require('./templates/header.html'),
```

Comment:
General comment, you can always run `./scripts/npm.sh run lint` locally to uncover any JS lint.